cmake/Format.cmake

```
@@ -55,10 +68,37 @@ set(
     ${PROJECT_SOURCE_DIR}/example/[^\.]*.cpp ${PROJECT_SOURCE_DIR}/example/[^\.]*.hpp
 )
 
+# Specifically trying to exclude external here - I'm not sure if there's a better way
+set(
+    PY_GLOBS
+    ${PROJECT_SOURCE_DIR}/scripts/[^\.]*.py
+    ${PROJECT_SOURCE_DIR}/tst/[^\.]*.py
+    ${PROJECT_SOURCE_DIR}/example/[^\.]*.py
```

```suggestion
    ${parthenon_SOURCE_DIR}/scripts/[^\.]*.py
    ${parthenon_SOURCE_DIR}/tst/[^\.]*.py
    ${parthenon_SOURCE_DIR}/example/[^\.]*.py
```

`PROJECT_SOURCE_DIR` depends on the latest `project()` call, so specifically using `parthenon_SOURCE_DIR` may be the more safe choice if Parthenon is included as subdirectory in a downstream code.

---

Good catch.